integration_testing/features/coach/coach-class-home.feature

```diff
@@ -0,0 +1,129 @@
+      And I see a list of learners with quiz progress
+
+  # Active but not started quizzes do not appear on Class home
+  Scenario: Review a not-started quiz 
+    Given there is a <quiz> quiz available
+      And there has been no learner progress on a <quiz> quiz
+    When I click the <quiz> quiz progress bar # Not yet implemented?
+    Then I am on <quiz> quiz page in the *Reports* tab
+      And I see high level quiz summary data
+      And I see a list of learners with quiz progress
+
+  Scenario: View all quizzes
+    Given that there may or may not be any quizzes available
+      When I press the *View all* button in the quizzes block
+      Then I am in *Coach > Reports > Quizzes* tab
+        And I see a list of all the class quizzes
+
+  Scenario: Review progress of a started lesson
+    Given there is a <lesson> lesson available
+      And there has been learner progress on a <lesson> lesson
+    When I click the <lesson> lesson progress bar # Not yet implemented?
+    Then I am on <lesson> lesson page in the *Reports* tab
+      And I see high level lesson summary data
+      And I see a list of learners with lesson progress
+
+  Scenario: Review a not-started lesson 
+    Given there is a <lesson> lesson available
+      And there has been no learner progress on a <lesson> lesson
+    When I click the <lesson> lesson progress bar # Not yet implemented?
+    Then I am on <lesson> lesson page in the *Reports* tab
+      And I see high level lesson summary data
+      And I see a list of lesson items with their completion status
+
+  Scenario: View all lessons
+    Given that there may or may not be any lessons available
+      When I press the *View all* button in the lessons block
+      Then I am in *Coach > Reports > Lessons* tab
+        And I see a list of all the class lessons      
+
+  Scenario: Review all class activity notifications
+    When I click *View all* in the *Class activity* block
+    Then I am on the *Class activity* page
+      And I see a history of all notifications
+      And I see filters for *Resource type* and *Progress type*
+
+  Scenario: Filter notifications by resource and progress   
+    Given that I am on the *Class activity* notifications page
+      When I open the *Resource type* filter
+      Then I see a list of resource options
+      When I open the *Progress type* filter
+      Then I see a list of progress options: all, completed, started, help needed
+
+  Scenario: Review details of a notification
+    Given that a notification just appeared in the *Class activity* notifications block
+      And I can see the notification displays the in-progress, completed, or help needed icon
+    When I click on the *Show* button for that notification
+    Then I am on the progress report page for the resource that emitted the notification
+
+# This does not make much sense as the user cannot arrive on the *Class home* if there are NO classes...?
+Feature: No classes in facility
```

Comment:
??? @khangmach ^^

Reply:
sorry I didn't clarify. CLASS HOME should be rename to HOME and the dash should only show notifications for resources users in the facility engaged with. Here's the Edulution mock JEssica worked on as reference: https://www.figma.com/file/D5wXhmWNGyt60SpGBgFucwZU/Coach-reports-0.12?node-id=719%3A0

Reply:
Thanks, I understand the context now, however I don't think we've ever tested the facility without classes before, at least I've never written any Gherkin scenarios for it... 😕 

I can wrap up something to test now, but what also worries me is that I don't see the `Home` string on Crowdin. @indirectlylit Is this change (_Class home_ to _Home_) even implemented?

Reply:
Valid concern! In my mind having the limited facility functionality is more important than incorporating the string change from class home > home. From a design perspective, keeping CLASS HOME string unchanged can even work in our benefit, to remind and indicate to the facility coach user that the dashboard works best/has more to show when classes are available

We can perhaps punt that verbiage change to another release as well.

Reply:
> Is this change (_Class home_ to _Home_) even implemented?

No - handling of notifications for the whole facility outside of classes was overlooked.

We have a few other implementation deficiencies and design inconsistencies with notifications:

* in implementation, 'all activity' items in class dashboard don't currently link to reports
* in implementation, 'activity' under learner reports is not yet implemented
* in design, 'activity' under group reports is missing even though logically it should be there
* in design, treating facilities with no classes differently than facilities with classes will result in broken UX: facility notifications would disappear if someone happens to make a class

I'll create a new issue to resolve all of these

Reply:
created new issue: #4928